release-notes/2023-07/SystemsConnectionMigration.md

```diff
@@ -0,0 +1,38 @@
+# **Migration plan for systems connectivity that is affected by the breaking change in the salt-master chart.**
+
+## The current release is introducing a breaking change regarding the systems connectivity. This means that any SLE <=0.14 release is affected by this change. The saltmaster service which manages the connectivity with the client systems has been replaced with a new implementation which causes all the connected clients to become disconnected. The next steps will describe how to re-approve and re-connect all the previously connected clients
+
+&nbsp;
+
+## **Before doing the upgrade of SLE**
+
+The old salt-master has a pair of RSA keys that are used for securing the communication. The public key is already cached on all the
+accepted clients. We need to persist these keys between all the SLE versions, otherwise the client systems will reject the new salt-master
+that would have a different public key.
+
+1. Open a terminal to the current "saltmaster-0" pod.
+2. Navigate to /etc/salt/pki/master
+3. Grab the master.pem (private key) and master.pub (public key) values.
+4. Set these values to the secrets definition object of the saltmaster chart in the top level secrets values yaml file. Make sure
+you use the "|" indicator because these are multiline strings.
+
+The new helm chart will create a kubernetes secret using these 2 keys and the pods will use it.
+
+&nbsp;
+
+## **After doing the upgrade of SLE**
+
+The expectation is that all the systems that prior the upgrade were connected (green/orange) are now disconnected and in the
```

Comment:
We likely need to reference the permissions needed by the user who is going to run the notebook for that operation to be successful.